libdnf/conf/ConfigRepo.cpp

```diff
@@ -145,7 +145,14 @@ ConfigRepo::Impl::Impl(Config & owner, ConfigMain & mainConfig)
     owner.optBinds().add("proxy_auth_method", proxy_auth_method);
     owner.optBinds().add("username", username);
     owner.optBinds().add("password", password);
-    owner.optBinds().add("protected_packages", protected_packages);
+
+    owner.optBinds().add("protected_packages", protected_packages,
+        [&](Option::Priority priority, const std::string & value){
+            if (priority >= protected_packages.getPriority())
```

Comment:
Why does `protected_packages` handle priority, while other list options, like `excludepkgs` don't?

Reply:
Thanks for the question. Thinking it over, it won't be 100% compatible with libdnf5 in either case - libdnf5 now includes sorting of loaded values based on their priority, which affects the clearing (empty) values. If the clearing happens at a different time, the result might change. However, if I remove the priority, it will at least be compatible with other libdnf append options.

I have removed the priority handling.